webscrapers/BCB_IF/classes/BancoCentral.py

```diff
@@ -0,0 +1,104 @@
+# Quantitative Finance
+# Insper
+# Authors: Ivan Zemella and Renan Pereira
+# Class
+
+import json,requests
+
+class BancoCentral:
+	
+	url_bancos 	= "https://www3.bcb.gov.br/ifdata/rest/arquivos?nomeArquivo=__trimestre__/cadastro__trimestre___1005.json&{}"
+	
+	url_valores = "https://www3.bcb.gov.br/ifdata/rest/arquivos?nomeArquivo=__trimestre__/dados__trimestre___1.json&{}"
+	
+	indicador_patrimonio_liquido = 78186
+	
+	indicador_lucro_liquido = 78187
+	
+	colecao = {}
+	
+	trimestres = ['201503','201506','201509','201512','201603','201606','201609','201612','201703','201706','201709','201712','201803','201806','201809','201812', '201903']
+
+
+	def obterTrimestre(self,trimestre):
+		self.obterBancos(trimestre)
+		self.obterValores(trimestre)
+		
+	
+	def obterBancos(self,trimestre):
+		
+		print('Capturing the banks for the quarter ...',trimestre)
+		
+		self.colecao[trimestre] = {}
+
+		resposta = requests.get(self.url_bancos.replace('__trimestre__',trimestre))
+		
+		dados = self.converterJson(resposta)
+		
+		i = 0
+		
+		while (i < len(dados)):
```

Comment:
You can change the `while` loop for the `pd.read_json` method. It will make things a lot faster and simpler.